### ISSUE TITLE
AirCrane somehow not showing in aircraft list

### ISSUE THREAD

**Issue by MV-DP** [wlbragg/AirCrane#4]
Hello,

Somehow the AirCrane doesn't show up in my aircraft list, I'm using FG 2018.2.2. And after download - master is removed (for example path Aircraft/Aircrane/aircrane-set.xml). Do I need to install other files in other folders to let it work with the new hauling system. 

Greets,

Martien van der Plas

**Comment by wlbragg** [comment]
I know, thanks for reminding me. I have looked and looked for the issue and can't figure it out as of yet. I will put some more effort into figuring out what the issue is.
Any help, hints or experience would be greatly welcome.

I just went and looked at my QT launcher aircraft list and I have all 4 repositories I have locally of different states and base repository structures of this aircraft, including the "installable" fgaddon version.
This is the first time I looked for it since I updated to next (fgfs, simgear, fgdata) two days ago. So magically it has appeared in the list now?
That is something I noticed before, it was at best inconsistent.
